LSE-61.tex

```diff
@@ -3642,6 +3642,786 @@ \subsubsection{No Limit on Data Access Centers}
+&
+10
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+dmEfdQueryTime
+} \\\hline
+Minimum number of simultaneous users querying the DM EFD.
+&
+5
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+dmEfdQueryUsers
+} \\\hline
+\end{parameters}
+
+\subsection{Image Access Performance Metrics}
+
+\subsubsection{Retrieval of postage stamp light curve images}
+
+\label{DMS-REQ-0375}
+\textbf{ID:} DMS-REQ-0375 (Priority: 2)
+
+\textbf{Specification:}
+Postage stamp cutouts, sized to match those in alert packets, of all observations of a single Object shall be retrievable within \textbf{postageStampRetrievalTime}, with \textbf{postageStampRetrievalUsers} simultaneous requests.
+
+\textbf{Discussion:}
+These postage stamps can be obtained from lossy compressed PVIs.
+
+\begin{parameters}
+Maximum time allowed for retrieving a set of postage stamp images of a single Object.
+&
+10
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+postageStampRetrievalTime
+} \\\hline
+Minimum number of simultaneous users retrieving a set of postage stamp images.
+&
+10
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+postageStampRetrievalUsers
+} \\\hline
+\end{parameters}
+
+\subsubsection{Retrieval of CCD-sized images}
+
+\label{DMS-REQ-0374}
+\textbf{ID:} DMS-REQ-0374 (Priority: 1b)
+
+\textbf{Specification:}
+A Processed Visit Image of a single CCD shall be retrievable using the VO SIAv2 protocol within \textbf{pviRetrievalTime} with \textbf{pviRetrievalUsers} simultaneous requests.
+
+\textbf{Discussion:}
+\begin{parameters}
+Minimum number of simultaneous users retrieving a single PVI image.
+&
+20
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+pviRetrievalUsers
+} \\\hline
+Maximum time allowed for retrieving a PVI image of a single CCD from a single visit
+&
+10
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+pviRetrievalTime
+} \\\hline
+\end{parameters}
+
+\subsubsection{Retrieval of focal-plane visit images}
+
+\label{DMS-REQ-0376}
+\textbf{ID:} DMS-REQ-0376 (Priority: 1b)
+
+\textbf{Specification:}
+All Processed Visit Images for a single visit, including mask and variance planes, shall be retrievable using the VO SIAv2 protocol within \textbf{allPviRetrievalTime} with \textbf{allPviRetrievalUsers} simultaneous requests.
+
+\begin{parameters}
+Maximum time allowed for retrieving all PVI images of a single visit.
+&
+60
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+allPviRetrievalTime
+} \\\hline
+Minimum number of simultaneous users retrieving all PVI images for a visit.
+&
+20
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+allPviRetrievalUsers
+} \\\hline
+\end{parameters}
+
+\subsubsection{Retrieval of focal-plane-sized images}
+
+\label{DMS-REQ-0373}
+\textbf{ID:} DMS-REQ-0373 (Priority: 2)
+
+\textbf{Specification:}
+A 10 square degree coadd, including mask and variance planes, shall be retrievable using the VO SIAv2 protocol within \textbf{fplaneRetrievalTime} with \textbf{fplaneRetrievalUsers} simultaneous requests.
```

Comment:
We need to think about whether we should really say "using SIAv2" here.  I think that may be presuming something.  It's probably actually a SODA request, unless the user's request really is for _the individual patch images_ (including overlaps) that cover the requested region.

Reply:
Oops. Sorry. Yes, "relevant VO standard"?